opensrp-goldsmith/src/main/java/org/smartregister/goldsmith/configuration/pnc/PncFormProcessor.java

```diff
@@ -35,13 +44,17 @@
     public HashMap<Client, List<Event>> extractEventClient(@NonNull String jsonString, @Nullable Intent data, @Nullable FormTag formTag) throws JSONException {
 
         AllSharedPreferences allSharedPreferences = ChwApplication.getInstance().getContext().allSharedPreferences();
-        EventClient registrationEventClient = JsonFormUtils.processRegistrationForm(allSharedPreferences, jsonString, Constants.TABLES.PREGNANCY_OUTCOME);
-        HashMap<Client, List<Event>> clientEventHashMap = new HashMap<>();
+        EventClient pncRegistrationEventClient = JsonFormUtils.processRegistrationForm(allSharedPreferences, jsonString, Constants.TABLES.PREGNANCY_OUTCOME);
```

Comment:
We have an issue where we get two clients in the clients table with similar first names. One of the client is missing essential attributes such as `attributes.opensrp_id`